tests/ui/methods.stderr

```diff
@@ -139,16 +139,27 @@ LL | |                    ).next();
    | |___________________________^
 
 error: called `is_some()` after searching an `Iterator` with find. This is more succinctly expressed by calling `any()`.
-  --> $DIR/methods.rs:265:13
+  --> $DIR/methods.rs:266:22
    |
 LL |     let _ = v.iter().find(|&x| *x < 0).is_some();
-   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `any(|x| x < 0)`
    |
    = note: `-D clippy::search-is-some` implied by `-D warnings`
-   = note: replace `find(|&x| *x < 0).is_some()` with `any(|x| *x < 0)`
 
 error: called `is_some()` after searching an `Iterator` with find. This is more succinctly expressed by calling `any()`.
-  --> $DIR/methods.rs:268:13
+  --> $DIR/methods.rs:267:20
+   |
+LL |     let _ = (0..1).find(|x| **y == *x).is_some(); // one dereference less
+   |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `any(|x| **y == *x)`
```

Comment:
this suggestion is still wrong

Reply:
as well as `any(|x| *x == 0)`

Reply:
Yes, it seems that the replacement does not work.

Reply:
it worked at this point https://github.com/rust-lang/rust-clippy/pull/4454#issuecomment-524998966 
Some of the changes did break it again. But I don't understand the code enough to get it back working

Reply:
Ah, I try to look into this, this evening. :+1: